applications/blackhole/src/modules/bh_call.erl

```diff
@@ -22,63 +22,64 @@
-                              ], EventJObj
-                             ) =:=
-        bh_context:account_id(Context).
+    NormJObj = kz_json:normalize_jobj(kz_json:set_value(<<"Binding">>, Binding, EventJObj)),
+    blackhole_data_emitter:emit(bh_context:websocket_pid(Context), event_name(EventJObj), NormJObj).
 
 -spec event_name(kz_json:object()) -> ne_binary().
 event_name(JObj) ->
     kz_json:get_value(<<"Event-Name">>, JObj).
 
--spec subscribe(bh_context:context(), ne_binary()) -> 'ok'.
+-spec subscribe(bh_context:context(), ne_binary()) -> {'ok', bh_context:context()}.
 subscribe(Context, <<"call.*.*">>) ->
     AccountId = bh_context:account_id(Context),
-    add_call_binding(AccountId, ?LISTEN_TO);
+    add_call_binding(AccountId, Context, ?LISTEN_TO),
+    {'ok', Context};
 subscribe(Context, <<"call.", Binding/binary>>) ->
     case binary:split(Binding, <<".">>, ['global']) of
         [Event, <<"*">>] ->
             AccountId = bh_context:account_id(Context),
-            add_call_binding(AccountId, [Event]);
+            add_call_binding(AccountId, Context, [Event]);
         _ ->
             blackhole_util:send_error_message(Context, <<"unmatched binding">>, Binding)
-    end;
+    end,
+    {'ok', Context};
 subscribe(Context, Binding) ->
-    blackhole_util:send_error_message(Context, <<"unmatched binding">>, Binding).
+    blackhole_util:send_error_message(Context, <<"unmatched binding">>, Binding),
+    {'ok', Context}.
 
--spec unsubscribe(bh_context:context(), ne_binary()) -> 'ok'.
+-spec unsubscribe(bh_context:context(), ne_binary()) -> {'ok', bh_context:context()}.
 unsubscribe(Context, <<"call.*.*">>) ->
     AccountId = bh_context:account_id(Context),
-    rm_call_binding(AccountId, ?LISTEN_TO);
+    rm_call_binding(AccountId, Context, ?LISTEN_TO),
+    {'ok', Context};
 unsubscribe(Context, <<"call.", Binding/binary>>) ->
     case binary:split(Binding, <<".">>, ['global']) of
         [Event, <<"*">>] ->
             AccountId = bh_context:account_id(Context),
-            rm_call_binding(AccountId, [Event]);
+            rm_call_binding(AccountId, Context, [Event]);
         _ ->
             blackhole_util:send_error_message(Context, <<"unmatched binding">>, Binding)
-    end;
+    end,
+    {'ok', Context};
 unsubscribe(Context, Binding) ->
-    blackhole_util:send_error_message(Context, <<"unmatched binding">>, Binding).
+    blackhole_util:send_error_message(Context, <<"unmatched binding">>, Binding),
+    {'ok', Context}.
 
--spec add_call_binding(ne_binary(), [ne_binary()]) -> ok.
-add_call_binding(_AccountId, []) -> ok;
-add_call_binding(AccountId, [Event | Events]) ->
+-spec add_call_binding(ne_binary(), bh_context:context(), [ne_binary()]) -> ok.
+add_call_binding(_AccountId, _Context, []) -> ok;
+add_call_binding(AccountId, Context, [Event | Events]) ->
+    blackhole_bindings:bind(<<"call.", Event/binary, ".*">>, ?MODULE, 'handle_event', Context),
```

Comment:
shouldn't we include the accountid in the binding key ? this way, this subscription will get notified for every account.
maybe that's why there was a check for the accountid.